src/redux/sagas/utils/updateColonyTokens.ts

```diff
@@ -39,6 +49,49 @@ export const getModifiedTokenAddresses = (
   return modifiedTokenAddress;
 };
 
+export const getPendingModifiedTokenAddresses = (
```

Comment:
The requirements here are different from the action.

Because the action completes synchronously, you can infer whether an address was added or removed by looking at whether it's currently present among the colony's tokens.

Because motions complete asynchrously, we need to be explicit about this.